src/logic/documents/scorecards.js

```diff
@@ -368,16 +379,23 @@ const scorecard = ({
     {
       margin: [25, 0, 0, 0],
       table: {
-        widths: ['*', 'auto', 'auto', ...(printStations ? ['auto'] : [])],
+        widths: [
+          '*',
+          getWidthOfBoxOrAuto(t('round'), 25),
+          getWidthOfBoxOrAuto(t('group'), 25),
+          ...(printStations ? [getWidthOfBoxOrAuto(t('station'), 25)] : []),
+        ],
         body: [
           columnLabels([
-            'Event',
-            'Round',
-            'Group',
-            ...(printStations ? ['Station'] : []),
+            t('eventLabel'),
+            { text: t('round'), alignment: 'center' },
+            { text: t('group'), alignment: 'center' },
+            ...(printStations
+              ? [{ text: t('station'), alignment: 'center' }]
+              : []),
```

Comment:
I would keep the current alignment.

```suggestion
            t('eventLabel'),
            t('round'),
            t('group'),
            ...(printStations ? [t('station')] : []),
```

Reply:
This is to prevent this (related to the previous choice though):
![skeudenn](https://github.com/user-attachments/assets/9c14f17c-663e-4f31-8ee0-f839613c2347)